roles/neutron-control/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
   tags: db-migrate
 
 - name: sync neutron database for lbaas
-  command: neutron-db-manage --service lbaas --config-file /etc/neutron/neutron.conf \
+  command: neutron-db-manage --subproject neutron-lbaas --config-file /etc/neutron/neutron.conf \
```

Comment:
why we need this change? We just completed QA and UAT of 3.1.1.3 and did not see any issues around neutron db sync